web/src/EnforcerListPage.js

```diff
@@ -23,6 +23,20 @@ import BaseListPage from "./BaseListPage";
 import PopconfirmModal from "./common/modal/PopconfirmModal";
 
 class EnforcerListPage extends BaseListPage {
+  constructor(props) {
+    super(props);
+    this.ref1 = createRef();
+    this.steps = [
+      {
+        title: "Enforcer List",
+        description: "In addition to the API interface for requesting enforcement of permission control, Casdoor also provides other interfaces that help external applications obtain permission policy information, which is also listed here.",
+        target: () => this.ref1,
+        nextButtonProps: {
+          children: "Go to \"Token List\"",
```

Comment:
children is too hard to organize. Can we put them together and organize them automatically?